packages/macaque/macaque.0.7.2/descr

```diff
@@ -0,0 +1 @@
+Macaque (Macros for Caml Queries) is a DSL for OCaml, wich produce SQL requests from a comprehension syntax. Macaque can build queries by from simpler components, using phantom types used to ensure safety.
```

Comment:
You may want that the first line is a short description (≤ 60 chars), the following lines giving possibly more details (wrapped to 80 chars please).